hathor/cli/run_node.py

```diff
@@ -89,6 +90,8 @@ def create_parser(self) -> ArgumentParser:
         parser.add_argument('--sentry-dsn', help='Sentry DSN')
         parser.add_argument('--enable-debug-api', action='store_true', help='Enable _debug/* endpoints')
         parser.add_argument('--enable-crash-api', action='store_true', help='Enable _crash/* endpoints')
+        parser.add_argument('--x-sync-bridge', action='store_true',
+                            help='Enable support for running both sync protocols. DO NOT ENABLE, IT WILL BREAK.')
```

Comment:
The full-node will not work because the sync-v2 is not implemented in this PR.

Reply:
But I see that we instance the same `NodeSyncManager` if we use sync-v1 or sync-v2 in the protocol, so it will work?

Reply:
It instantiates a sync-v1 manager where there would be a sync-v2. It will _work_ as long as we don't change that between different versions. But we precisely want to change that and two nodes using `--x-sync-bridge --unsafe-mode=mainnet` with different versions of the code _will not_ be able sync because they would be using different protocols thinking they are both using sync-v2.